script/configure-web-proxy

```diff
@@ -36,11 +42,17 @@ sed -i -e 's/^.*httpsonly.*$/httpsonly = 0/g' /etc/openqa/openqa.ini
 if [[ $web_proxy == "nginx" ]]; then
     echo "Setting up nginx"
     sed "s/openqa.example.com/$(hostname)/" /etc/nginx/vhosts.d/openqa.conf.template > /etc/nginx/vhosts.d/openqa.conf
+    # IPv6 uses [::]:80 syntax instead of just a port
+    if [[ -n "$web_port" ]]; then
+        sed -Ei "s@(.*listen.*)80@\1${web_port}@g" /etc/nginx/vhosts.d/openqa.conf
+        sed -i "s/listen.*$/listen $web_port;/" /etc/nginx/nginx.conf
```

Comment:
I think we need the same as for the openqa.conf here. WDYT about merging both into one call:
```suggestion
       sed -Ei "s@(.*listen.*)80@\1${web_port}@g" /etc/nginx/vhosts.d/openqa.conf /etc/nginx/nginx.conf
```